draft-ietf-jose-json-web-proof.md

```diff
@@ -282,6 +282,35 @@ containing a StringOrURI value.
 Its definition is intentionally parallel to the `iss` claim defined in [@!RFC7519].
 Use of this Header Parameter is OPTIONAL.
 
+### "aud" (Audience) Header Parameter {#audDef}
+
+The `aud` (audience) Header Parameter
+identifies the recipients that the JWP is intended for.
+Each principal intended to process the JWP MUST identify itself
```

Comment:
I've always thought the "identify itself with a value" terminology was really confusing, since the verifier (in this case) does not get to set `aud`. "Confirm its identity is represented by at least one value" would have been so much better - but I suppose we are stuck with it for consistency.

Reply:
It's because especially in SAML deployments, the audiences are often abstract things like urn:edu:nternet2:federation.